spec/examples/full_email.txt

```diff
@@ -1,4 +1,4 @@
-http://localhost/home	16 December 2015
+[image]	16 December 2015
```

Comment:
Why is this converted to `[image]`?  That doesn't seem right.

Reply:
Nevermind.  I didn't see where this originated in `full_email.html`.

Reply:
Thanks for pointing that out. In the html, this is a link-wrapped image.

https://github.com/Yesware/html2text_ruby/blob/master/spec/examples/full_email.html#L29

Lexis asked that urls be stripped from link-wrapped images.